src/components/Global/Logo/Logo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Image from 'next/image';
+import logo from '../../../../public/assets/logo.jpg';
+import logoStyles from './Logo.module.scss';
+
+type LogoProps = {
+  showMark: boolean;
+  showType: boolean;
+};
+
+const Logo = ({ showMark, showType }: LogoProps): JSX.Element => {
```

Comment:
It's not necessary to add JSX.Element since it's usually included for clarification in a .ts/.js file, but we are using .tsx file.